Formula/clap-host.rb

```diff
@@ -0,0 +1,24 @@
+class ClapHost < Formula
+  desc "CLAP example host"
+  homepage "https://github.com/free-audio/clap-host"
+  url "https://github.com/free-audio/clap-host", using: :git, tag: "1.0.1", revision: "4602d26b91a526ac80933bdc9f6fcafdf2423bbf"
```

Comment:
This should be enough.
```suggestion
  url "https://github.com/free-audio/clap-host", 
        tag: "1.0.1",
        revision: "4602d26b91a526ac80933bdc9f6fcafdf2423bbf"
```